services/QuillLMS/lib/rule_feedback_history.rb

```diff
@@ -26,6 +26,30 @@ def self.exec_query(conjunction:, activity_id:)
         rules = Comprehension::Rule.find_by_sql(sql)
     end
 
+    def self.feedback_history_to_json(f_h)
+        {
+            response_id: f_h.id,
+            datetime: f_h.updated_at,
+            entry: f_h.entry,
+            highlight: f_h.metadata.class == Hash ? f_h.metadata['highlight'] : '',
+            view_session_url: 'Not yet available'
+        }
```

Comment:
I think we need a `strength` attribute here, because we need that data for determining what to show on the frontend. I'm sorry for not calling that out on your original example payload - I think we hadn't yet decided whether or not "strength" was making it into this version of the page.

Reply:
addressed!